.github/workflows/trigger-load-tests.yml

```diff
@@ -15,49 +15,16 @@ on:
     - cron: '30 0 * * *'
 
 jobs:
-  invokeLoadTests:
-    runs-on: ubuntu-latest
-    if: github.repository_owner == 'ballerina-platform'
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Process cluster name
-        id: processClusterName
-        run: |
-          clusterName="${{github.event.inputs.clusterName}}"
-          if [ "$clusterName" = "" ]; then
-            clusterName="io-perf-cluster-test"
-          fi
-          printf "clusterName=%s" $clusterName >> $GITHUB_OUTPUT
-      - name: Get latest ballerina runtime artifact
-        id: setRuntimeUrl
-        shell: bash
-        run: |
-          printf "runtimeUrl=%s" $(curl -H "Accept: application/vnd.github.v3+json" \
-            https://api.github.com/repos/ballerina-platform/module-ballerina-io/actions/artifacts | \
-            jq '[.artifacts[] | select(.name=="ballerina-runtime" and .expired==false)]|max|.archive_download_url') >> $GITHUB_OUTPUT
-      - name: Format test names
-        id: formatTestNames
-        shell: bash
-        run: |
-          tests="${{github.event.inputs.tests}}"
-          if [ ! "$tests" = "" ]; then
-            testsInput=(${tests//,/ })
-            testsCount=${#testsInput[@]}
-            testsOutput=""
-            for (( i=0; i<$testsCount; i++ ));
-            do
-                testsOutput+=\\'"'${testsInput[$i]}\\'"',
-            done
-            printf "testNames=%s" [${testsOutput:0:-1}] >> $GITHUB_OUTPUT
-          else
-            printf "testNames=%s" $tests >> $GITHUB_OUTPUT
-          fi
-      - name: Invoke standard library load test workflow
-        uses: benc-uk/workflow-dispatch@v1
-        with:
-          workflow: Stdlib Workflow
-          repo: ballerina-platform/ballerina-performance-cloud
-          ref: refs/heads/main
-          token: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          inputs: '{ "repo-name": "module-ballerina-io", "tests": "${{ steps.formatTestNames.outputs.testNames }}", "clusterName": "${{ steps.processClusterName.outputs.clusterName }}", "dispatchType": "io-load-test" }'
+  call_stdlib_trigger_load_test_workflow:
+    name: Run StdLib Load Test Workflow
+    if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/trigger-load-tests-template.yml@main
+    with:
+      repo_name: 'module-ballerina-io'
+      runtime_artifacts_url: 'https://api.github.com/repos/ballerina-platform/module-ballerina-io/actions/artifacts'
+      dispatch_type: 'io-load-test'
+      cluster_name: ${{ inputs.clusterName }}
+      tests: ${{ inputs.tests }}
+      branch: ${{ inputs.branch }}
+      secrets:
```

Comment:
indentation issue. `secrets` should align with `with:`

Reply:
updated in [afe7df1](https://github.com/ballerina-platform/module-ballerina-io/pull/923/commits/afe7df10f171d4eb21ccdc266783547a5939456e)